libdirections-onboard/src/main/java/com/mapbox/navigation/route/onboard/internal/OnboardRouterFiles.kt

```diff
@@ -0,0 +1,41 @@
+package com.mapbox.navigation.route.onboard.internal
+
+import android.content.Context
+import com.mapbox.base.common.logger.Logger
+import com.mapbox.base.common.logger.model.Message
+import com.mapbox.navigation.base.options.OnboardRouterOptions
+import java.io.File
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
+
+internal class OnboardRouterFiles(
+    val applicationContext: Context,
+    val logger: Logger
+) {
+
+    suspend fun absolutePath(options: OnboardRouterOptions): String? = withContext(Dispatchers.IO) {
```

Comment:
Why not relying on `ThreadController.IODispatcher`? We do that for example in https://github.com/mapbox/mapbox-navigation-android/blob/23f85cbea4f6e38a0232763608fcaaa8c1d88b45/libdirections-onboard/src/main/java/com/mapbox/navigation/route/onboard/internal/MapboxOnboardRouter.kt#L194

Reply:
yeah what's the reason for having ThreadController.IODispatchers again? it restricts us to 2 threads, where Dispatchers.IO uses the number of available processors

Reply:
maybe we can test it with mobile metrics one day. ThradController.IODispatchers will give us that